### PR TITLE
Aden 3574

### PR DESCRIPTION
**Pull request by bognix**
DON'T MERGE - THIS CODE IS ONLY NEEDED TO GATHER DATA

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_Interstital() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng02.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+  }
+
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_FloorAdhesion() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_Interstital() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues:
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testsuites/testSuite.xml [hunk 3]
```diff
@@ -60,6 +60,7 @@
             <class name="com.wikia.webdriver.testcases.adstests.TestTopWamWikis"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestUap"/>
             <class name="com.wikia.webdriver.testcases.adstests.TestUrlBuilder"/>
+            <class name="com.wikia.webdriver.testcases.adstests.TestHorseSlots"/>
```
Comment (tomasznapieralski) [inline src/test/java/com/wikia/webdriver/testsuites/testSuite.xml]: What are horse slots? (końskie gniazda) :D

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstital() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng01.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstital() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng01.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/FloorAdhesion");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-s6.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/FloorAdhesion");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_Interstital", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstital() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng02.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+    try {
+      Thread.sleep(10000);
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: is that sleep the best way to end that test ?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng02.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/FloorAdhesion");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/gpt/INVISIBLE_HIGH_IMPACT_2");
+    try {
+      Thread.sleep(10000);
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: is that sleep the best way to end that test ?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java (hunk 4 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java (hunk 5 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java (hunk 6 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java (hunk 7 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
+    driver.manage().window().maximize();
+    driver.get("http://aga.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/FloorAdhesion");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("google_ads_iframe_/5441/wka.gaming/_aga//article/gpt/INVISIBLE_HIGH_IMPACT_2_0__container__");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstitial() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 13]
```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstitial() {
+    driver.manage().window().maximize();
+    driver.get("http://aga.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("google_ads_iframe_/5441/wka.gaming/_aga//article/gpt/INVISIBLE_HIGH_IMPACT_2_0__container__");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_FloorAdhesion() {
+    driver.manage().window().maximize();
+    driver.get("http://aga.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/FloorAdhesion");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("google_ads_iframe_/5441/wka.gaming/_aga//article/gpt/INVISIBLE_HIGH_IMPACT_2_0__container__");
+    try {
+      Thread.sleep(15000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 16]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstitial() {
+    driver.manage().window().maximize();
+    driver.get("http://aga.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("google_ads_iframe_/5441/wka.gaming/_aga//article/gpt/INVISIBLE_HIGH_IMPACT_2_0__container__");
+    try {
+      Thread.sleep(15000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 17]
```diff
@@ -0,0 +1,34 @@
...
+//    ads.waitForElementPresent("google_ads_iframe_/5441/wka.gaming/_aga//article/gpt/INVISIBLE_HIGH_IMPACT_2_0__container__");
+//    try {
+//      Thread.sleep(10000);
+//    } catch (InterruptedException e) {
+//      e.printStackTrace();
+//    }
+//  }
+//
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstitial() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 18]
```diff
@@ -0,0 +1,34 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+//  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+//  public void TestHorseSlots_FloorAdhesion() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 19]
```diff
@@ -0,0 +1,34 @@
...
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstitial() {
+    driver.manage().window().maximize();
+    driver.get("http://sandbox-adeng01.project43.wikia.com/wiki/SyntheticTests/Slots/InvisibleHighImpact/Interstitial");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/mobile/INVISIBLE_HIGH_IMPACT_2");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* CRITICAL 1 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 20]
```diff
@@ -0,0 +1,43 @@
...
+//    try {
+//      Thread.sleep(10000);
+//    } catch (InterruptedException e) {
+//      e.printStackTrace();
+//    }
+//  }
+//
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 5)
+  public void TestHorseSlots_Interstitial() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* CRITICAL 1 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 21]
```diff
@@ -0,0 +1,43 @@
...
+    driver.get("http://bogna.wikia.com/wiki/A");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.gaming/_bogna//article/mobile/MOBILE_TOP_LEADERBOARD");
+    WebElement link = driver.findElement(By.cssSelector("a[title='B']"));
+    link.click();
+    ads.waitForElementPresent("wikia_gpt/5441/wka.gaming/_bogna//article/mobile/MOBILE_TOP_LEADERBOARD");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* CRITICAL 1 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 22]
```diff
@@ -0,0 +1,43 @@
...
+    driver.get("http://project43.wikia.com/wiki/A");
+    AdsBaseObject ads = new AdsBaseObject(driver);
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/mobile/MOBILE_TOP_LEADERBOARD");
+    WebElement link = driver.findElement(By.cssSelector("a[title='B']"));
+    link.click();
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/mobile/MOBILE_TOP_LEADERBOARD");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* CRITICAL 1 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* CRITICAL 1 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* CRITICAL 1 critical

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 23]
```diff
@@ -0,0 +1,48 @@
...
+//    try {
+//      Thread.sleep(10000);
+//    } catch (InterruptedException e) {
+//      e.printStackTrace();
+//    }
+//  }
+//
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  @Test(groups = {"TestHorseSlots_Interstitial", "TestHorseSlots"}, invocationCount = 10)
+  public void TestHorseSlots_Interstitial() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 24]
```diff
@@ -0,0 +1,48 @@
...
+    } catch (InterruptedException e) {
+      e.printStackTrace();
+    }
+    WebElement link = driver.findElement(By.cssSelector("a[title='B']"));
+    link.click();
+    ads.waitForElementPresent("wikia_gpt/5441/wka.life/_project43//article/mobile/MOBILE_TOP_LEADERBOARD");
+    try {
+      Thread.sleep(10000);
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java [hunk 25]
```diff
@@ -0,0 +1,48 @@
...
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.annotations.Test;
+
+public class TestHorseSlots extends NewTestTemplate {
+
+//  @Test(groups = {"TestHorseSlots_FloorAdhesion", "TestHorseSlots"}, invocationCount = 5)
+//  public void TestHorseSlots_FloorAdhesion() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: MAJOR This block of commented-out lines of code should be removed. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
@bognix hey, is this PR still valid? Maybe we can close it?

File: src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java (hunk 24 above)
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/adstests/TestHorseSlots.java]: CRITICAL Use a logger to log this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1148)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* CRITICAL 2 critical
* MAJOR 2 major

Watch the comments in this conversation to review them.

**Pull request closed without merging**